src/utils_project.js

```diff
@@ -269,9 +276,15 @@ go.utils_project = {
             'state_end_optout'
         ];
 
+        var voice_url_check = voice_url.length === 1
+            // if length is 1, check the only url in array
+            ? voice_url[0]
+            // else in error case, length > 1, check second url in array
+            : voice_url[1];
```

Comment:
I would think it might be more helpful to check both URLs here - what if the error state recording is missing / misplaced?